README.md

```diff
@@ -14,7 +14,9 @@
 
 1. Run `npm install -g urcli`.
 1. Get a new token from the API Access link in the [Reviewer Dashboard](https://review.udacity.com/#!/submissions/dashboard).
-<img src="http://i.imgur.com/QH7onbk.png" alt="Token retrieval" width="500px">
+
+    <img src="http://i.imgur.com/QH7onbk.png" alt="Token retrieval" width="500px">
```

Comment:
If you decide to do another PR then just explain quickly why you think the change is needed and I'll be happy to merge it.